ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/cmake/tasks/main.yml

```diff
@@ -31,7 +31,7 @@
     checksum: sha256:8f864e9f78917de3e1483e256270daabc4a321741592c5b36af028e72bff87f5
   when:
     - (cmake_installed.rc != 0 ) or (cmake_installed.rc == 0 and cmake_version.stdout is version_compare(cmakeVersion, operator='lt') )
-    - (ansible_architecture == "x86_64") or (ansible_architecture == "ppc64le") or (ansible_architecture == "s390x")
+    - (ansible_architecture == "x86_64") or (ansible_architecture == "ppc64le") or (ansible_architecture == "s390x") or (ansible_architecture == "aarch64")
```

Comment:
This covers all linux architectures other than armv7l (arm32) right? So I would suggest modifying this conditional to say all architectures other than armv7l to make it easier to scale this.

However it's probably worth looking at whether we even need an arm32 specific clause here. Perhaps we need to check whether cmake 3.11.4 is already in the package repositories for various distributions and if so add it into the Common role where we install stuff through the package manager. If we do that in a way that installs on armv7l/arm32 then I suspect the clause at the start of this role which checks for cmake being installed would trap that and mean we don't need all these conditionals.